### ISSUE TITLE
[LroExtension & GetOperation200] Not handle the GET polling API - false positive 

### ISSUE THREAD

**Issue by ms-henglu** [Azure/azure-openapi-validator#549]
**Describe the bug**

This is a representation of the location header status polling API and the linter rules haven't accounted for it.

```json
 "/providers/Microsoft.Subscription/subscriptionOperations/{operationId}": {
      "get": {
        "description": "Get the status of the pending Microsoft.Subscription API operations.",
        "operationId": "SubscriptionOperation_Get",
        "x-ms-examples": {
          "getPendingSubscriptionOperations": {
            "$ref": "./examples/getSubscriptionOperation.json"
          }
        },
        "responses": {
          "200": {
            "description": "Successful completion of the asynchronous operation",
            "schema": {
              "$ref": "#/definitions/SubscriptionCreationResult"
            }
          },
          "202": {
            "description": "Accepted. Subscription update is in progress.",
            "headers": {
              "Location": {
                "description": "The URL where the status of the asynchronous operation can be checked.",
                "type": "string"
              },
              "Retry-After": {
                "description": "The amount of delay to use while the status of the operation is checked. The value is expressed in seconds.",
                "type": "integer",
                "format": "int64"
              }
            }
          },
          "default": {
            "description": "Error response describing why the operation failed.",
            "schema": {
              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
            }
          }
        },
        "parameters": [
          {
            "name": "operationId",
            "in": "path",
            "description": "The operation ID, which can be found from the Location field in the generate recommendation response header.",
            "required": true,
            "type": "string"
          },
          {
            "$ref": "#/parameters/apiVersionParameter"
          }
        ]
      }
    },
```
**To Reproduce**

**Expected behavior**

**Screenshots**
<img width="1126" alt="image" src="https://github.com/Azure/azure-openapi-validator/assets/79895375/2f007a8c-50d8-416f-ad4c-fa813a3e0970">

**Desktop (please complete the following information):**

**Additional context**
https://github.com/Azure/azure-rest-api-specs/pull/24395/files
